src/passwordless/email_sent_confirmation.jsx

```diff
@@ -13,7 +13,7 @@ class ResendLink extends React.Component {
   render() {
     const { label, onClick } = this.props;
     return(
-      <a className="auth0-lock-resend-link" href="" onClick={onClick}>
+      <a className="auth0-lock-resend-link" href="#" onClick={onClick}>
```

Comment:
What about event.preventDefault()? Doesn't it work? also a `button(type="button")` might do the trick (although I'm not sure about side effects). Anyway, this approach is also ok to me, a little bit weird from a puristic semantic point of view.